README.md

```diff
@@ -1 +1,6 @@
 # Juntos Frontend
+
+### How to use Project.
```

Comment:
Why are we using a level 3 heading, instead of level 2, will there be a parent section?

Reply:
gave it heading 2. No there wont be any such parent section